sklearn/decomposition/_sparse_pca.py

```diff
@@ -105,10 +165,14 @@ class SparsePCA(TransformerMixin, BaseEstimator):
     n_iter_ : int
         Number of iterations run.
 
-    mean_ : array, shape (n_features,)
+    mean_ : array, shape (n_features, )
```

Comment:
Please revert this line, a one element tuple is indeed `(n_features,)` without the space